sos/report/plugins/opensvc.py

```diff
@@ -59,6 +59,21 @@ def setup(self):
         self.add_dir_listing('/var/lib/opensvc', recursive=True)
         self.get_status('vol')
         self.get_status('svc')
+        
+        pid_file = "/var/lib/opensvc/osvcd.pid"
+        try:
+            with open(pid_file, 'r') as file:
+                pid = file.read().strip()
+                if not pid:
+                    self._log_debug("%s is empty" % pid_file)
+                    return
+                if not pid.isdigit():
+                    self._log_debug("Invalid PID in %s: %s" % (pid_file, pid))
+                    return
+                self._log_debug("Collecting threads for PID: %s" % pid)
```

Comment:
We don't need a debug here. If the file is collected, then this is already logged in sos.log.